### DISCUSSION TITLE
Package microtype warning

### DISCUSSION THREAD

**Discussion by rmfmarinho** [joaomlourenco/novathesis#438] [General]
Boa tarde,

Gostava de agradecer primeiro ao professor por todo o material que fornece e apoio contínuo que dá aos alunos.

Tenho uma dúvida que se arrasta há algum tempo, tinha alguns warnings que consegui resolver mas sobrou o seguinte:

"Package microtype Warning: I cannot find a spacing list for font `TS1/zplTLF/m/n/10.95' (context: `nonfrench'). Switching off spacing for this font on input line 70."

O erro reencaminha-me para um itemize simples no capitulo 1, igual a bastantes outros que tenho ao longo da dissertação.

Obrigado pela atenção!

Rodrigo Marinho 62233

**Comment by joaomlourenco** [comment]
Dear Rodrigo,

Please use english in this channel… ok?  Thank you!

Concerning your question, microtype implements a very fine tuning positioning of characters, leading the output closer to “perfection”. 
I think I can "read and understand" the warning, but I have no ideia on how to solve the problem.  But I am sure it can be safely ignored, and the impact is minimal (very very tiny details) in the text presentation.

**Reply by rmfmarinho** [reply]
Okay thank you
